odoo_test_xmlrunner/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Unittest xUnit reports",
+    "version": "17.0.1.0.0",
+    "depends": ["base"],
+    "author": "martin.deconinck@smile.fr, Odoo Community Association (OCA)",
```

Comment:
Here it should be the company:

```suggestion
    "author": "Smile, Odoo Community Association (OCA)",
```